.github/workflows/benchmarks-report.yaml

```diff
@@ -2,7 +2,7 @@ name: Report Benchmark Results
 
 on:
   workflow_run:
-    workflows: ['Benchmarks Test']
+    workflows: [Benchmarks]
```

Comment:
Nice – so this should hook into the modified Benchmarks action `requested` and `completed`.